envSchema.js

```diff
@@ -66,7 +66,7 @@ const properties = {
     minimum: 1,
   },
   TRUSTED_PROXIES: { type: 'string', default: '10.0.0.0/8,172.16.0.0/12,192.168.0.0/16' },
-  KMS_PROVIDER: { type: 'string', enum: ['gcp', 'local', 'none'], description: 'Master key manager', default: 'none' },
+  OPEN_API_SPECIFICATION: { type: 'string', enum: ['swagger', 'openapi'], description: 'OpenAPI specification used to expose Swagger', default: 'swagger' },
```

Comment:
```suggestion
  OPEN_API_SPECIFICATION: { type: 'string', enum: ['swagger', 'openapi'], description: 'defines which type of API documentation the service should generate and expose, that is either OpenAPI Specification or Swagger', default: 'swagger' },
```
What do you think about this version?